### ISSUE TITLE
bunx nx serve application on bun 1.1.36

### ISSUE THREAD

**Issue by kdriedger** [oven-sh/bun#15292]
### How can we reproduce the crash?

if this is deemed important, more time can be dedicated...  it's a proprietary angular app.  Saved the bun.lockb file if needed

### Relevant log output

```shell
NX   Failed to process project graph. Run "nx reset" to fix this. Please report the issue if you keep seeing it. See errors below.

Failed to process project graph. Run "nx reset" to fix this. Please report the issue if you keep seeing it.
  AggregateCreateNodesError: An error occurred while processing files for the nx/js/dependencies-and-lockfile plugin.
    - bun.lockb: Command failed: bun C:\Users\KDRIEDGE\source\repos\app\Web\bun.lockb
  ============================================================
  Bun v1.1.36 (ededc168) Windows x64
  Windows v.win10_fe
  CPU: sse42 avx avx2 avx512
  Args: "bun" "C:\Users\KDRIEDGE\source\repos\app\Web\bun.lockb"
  Elapsed: 10ms | User: 15ms | Sys: 15ms
  RSS: 23.52MB | Peak: 23.52MB | Commit: 54.16MB | Faults: 5903

  panic(main thread): incorrect alignment
  oh no: Bun has crashed. This indicates a bug in Bun, not your code.

  To send a redacted crash report to Bun's team,
  please file a GitHub issue using the link below:

   https://bun.report/1.1.36/wa1ededc16AA0mi3C6i9rM8+6rM2/q3Hg23oF28g9Fulj+F48hBi8I4ykwkDA0eNrLzEvOLypKTS5RSMzJTM/LTc0rAQBMcwep


      at createNodesFromFiles (C:\Users\KDRIEDGE\source\repos\app\Web\node_modules\nx\src\project-graph\plugins\utils.js:58:15)
      at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
      at async LoadedNxPlugin.createNodes.<computed> (C:\Users\KDRIEDGE\source\repos\app\Web\node_modules\nx\src\project-graph\plugins\internal-api.js:41:28) 
      at async LoadedNxPlugin.createNodes.<computed> (C:\Users\KDRIEDGE\source\repos\app\Web\node_modules\nx\src\project-graph\plugins\internal-api.js:41:28)  
      at async Promise.all (index 1)
      at async processFilesAndCreateAndSerializeProjectGraph (C:\Users\KDRIEDGE\source\repos\app\Web\node_modules\nx\src\daemon\server\project-graph-incremental-recomputation.js:148:43)
      at async getCachedSerializedProjectGraphPromise (C:\Users\KDRIEDGE\source\repos\app\Web\node_modules\nx\src\daemon\server\project-graph-incremental-recomputation.js:46:16)
      at async handleRequestProjectGraph (C:\Users\KDRIEDGE\source\repos\app\Web\node_modules\nx\src\daemon\server\handle-request-project-graph.js:12:24)      
      at async handleResult (C:\Users\KDRIEDGE\source\repos\app\Web\node_modules\nx\src\daemon\server\server.js:130:16)
      at async handleMessage (C:\Users\KDRIEDGE\source\repos\app\Web\node_modules\nx\src\daemon\server\server.js:89:9)
      at async C:\Users\KDRIEDGE\source\repos\app\Web\node_modules\nx\src\daemon\server\server.js:55:9
```

### Stack Trace (bun.report)

Bun v1.1.36 ([`ededc16`](<https://github.com/oven-sh/bun/tree/ededc168cf07afcdbe5ca34495c58df435ea526d>)) on windows x86_64 [AutoCommand]

**panic**: incorrect alignment

- [`dependency.zig:1072`](<https://github.com/oven-sh/bun/blob/ededc168cf07afcdbe5ca34495c58df435ea526d/src/install/dependency.zig#L1072>): `parseWithTag`
- [`dependency.zig:416`](<https://github.com/oven-sh/bun/blob/ededc168cf07afcdbe5ca34495c58df435ea526d/src/install/dependency.zig#L416>): `toVersion`
- [`lockfile.zig:5869`](<https://github.com/oven-sh/bun/blob/ededc168cf07afcdbe5ca34495c58df435ea526d/src/install/lockfile.zig#L5869>): `load`
- [`lockfile.zig:6123`](<https://github.com/oven-sh/bun/blob/ededc168cf07afcdbe5ca34495c58df435ea526d/src/install/lockfile.zig#L6123>): `load`
- [`lockfile.zig:263`](<https://github.com/oven-sh/bun/blob/ededc168cf07afcdbe5ca34495c58df435ea526d/src/install/lockfile.zig#L263>): `loadFromDisk`
- [`lockfile.zig:1241`](<https://github.com/oven-sh/bun/blob/ededc168cf07afcdbe5ca34495c58df435ea526d/src/install/lockfile.zig#L1241>): `print`
- [`cli.zig:2105`](<https://github.com/oven-sh/bun/blob/ededc168cf07afcdbe5ca34495c58df435ea526d/src/cli.zig#L2105>): `start`
- [`cli.zig:63`](<https://github.com/oven-sh/bun/blob/ededc168cf07afcdbe5ca34495c58df435ea526d/src/cli.zig#L63>): `start`
- [`main.zig:49`](<https://github.com/oven-sh/bun/blob/ededc168cf07afcdbe5ca34495c58df435ea526d/src/main.zig#L49>): `main`
- [`exe_common.inl:78`](<https://github.com/oven-sh/bun/blob/ededc168cf07afcdbe5ca34495c58df435ea526d/src/src/startup/exe_common.inl#L78>): `invoke_main`

<sub>Sentry Issue: <strong><a href="https://bun-p9.sentry.io/issues/6067098973/">BUN-8FJ</a></strong></sub>

**Comment by santiagof4** [comment]
I have the same issue. Downgrading to 1.1.34 is the only solution right now. 1.1.35 has the same issue.

**Comment by santiagof4** [comment]
I can confirm that this issue is not present in v1.1.37 🙂

**Comment by Jarred-Sumner** [comment]
Fixed in v1.1.37
